### ISSUE TITLE
Ask / warn user to setup a venv before starting notebooks

### ISSUE THREAD

**Issue by natct10** [SubstraFoundation/distributed-learning-contributivity#208]
As the notebooks are installing several dependencies, it would be great to add a note about creating a python venv before running it.
What do you about it? Is it obvious or nice to have?

**Comment by RomainGoussault** [comment]
this is not obvious ! (not for me at least ! :))

**Comment by bowni** [comment]
@arthurPignet a good element to add in the package documentation on which you are working with PR #233
